installing/install_config/installing-customizing.adoc

```diff
@@ -0,0 +1,26 @@
+[id="installing-customizing"]
+= Customizing nodes
+include::modules/common-attributes.adoc[]
+:context: installing-customizing
+
+toc::[]
+
+While we discourage making changes directly to {product-title} nodes,
```

Comment:
Consider rewriting the first sentence to avoid the use of "we":
Although directly making changes to {product-title} nodes is discouraged, there are times when it  is necessary . . .

Reply:
Done